src/style.css

```diff
@@ -0,0 +1,138 @@
+@font-face {
+  font-family: Avenir-Book, serif;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  font-family: Avenir-Book, sans-serif;
+}
+
+form[data-qa="big"] {
+  margin-top: 20px;
+  margin-left: 8px;
+  margin-bottom: 20px;
+  margin-right: 8px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  height: 70px;
+  display: flex;
+}
+
+.search__main-button {
+  padding: 0;
+  margin-top: 2px;
+  margin-left: 25px;
+  margin-right: 18px;
+  border: none;
+  background-color: #fff;
+}
+
+.search__main-img {
+  height: 19px;
+  width: 19px;
+}
+
+input[data-qa="keypress"] {
+  padding: 0;
+  border: none;
+  height: 100%;
+  width: 100%;
+  font-size: 16px;
+  line-height: 22px;
+  font-weight: 900px;
+}
+
+input[data-qa="keypress"]::placeholder {
+  font-weight: 400;
+  color: #3d4e61;
+  line-height: 22px;
+}
+
+form[data-qa="big"]:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+form[data-qa="big"]:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-weight: 900;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+input[data-qa="keypress"]:focus {
+  font-weight: 900;
+  font-size: 16px;
+  line-height: 22px;
+  border: none;
+  outline: none;
+}
+
+form[data-qa="small"] {
+  margin-left: 8px;
+  margin-right: 8px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  height: 42px;
+  display: flex;
+}
+
+.search__submain-button {
+  margin-left: 12px;
+  margin-right: 9px;
+  margin-top: 2px;
+  padding: 0;
+  border: none;
+  background-color: #fff;
+}
+
+.search__submain-img {
+  height: 11px;
+  width: 11px;
+}
+
+.search__submain-input {
+  border: none;
+  height: 100%;
+  width: 100%;
+  font-size: 14px;
+}
+
+.search__submain:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.search__submain:focus {
```

Comment:
you are duplicating a lot of code create main class for search-bars with common styles, and create additional classes like `search-bar--small/big` to malke them different